### PR TITLE
NOISSUE - Update Docs

### PR DESCRIPTION
**Pull request by WashingtonKK**
Signed-off-by: WashingtonKK <washingtonkigan@gmail.com>

### CODE REVIEW

File: docs/manager.md [hunk 1]
```diff
@@ -7,3 +7,217 @@ Vsock is used to send agent events from the computation in the agent to the mana
 ## Manager <> Agent
 
 Agent runs a gRPC server, and CLI is a gRPC client of agent. The manager sends the computation to the agent via gRPC and the agent runs the computation while sending evnets back to manager on the status. The manager then sends the events it receives from agent via vsock through gRPC.
+
+## Setup and test manager <> agent
```
Comment (drasko) [inline docs/manager.md:11]: Capitalize words in titles

File: docs/manager.md [hunk 2]
```diff
@@ -7,3 +7,217 @@ Vsock is used to send agent events from the computation in the agent to the mana
...
+    -kernel $KERNEL \
+    -append "earlyprintk=serial console=ttyS0" \
+    -initrd $INITRD \
+    -nographic \
+    -monitor pty \
+    -monitor unix:monitor,server,nowait
+```
+Once the VM is booted press enter and on the login use username `root`.
+
+#### Build and run Agent
```
Comment (drasko) [inline docs/manager.md:88]: Capitalize

**Review by drasko — CHANGES_REQUESTED** [review]

File: docs/agent.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
+The agent is responsible for the life cycle of the computation, i.e., running the computation and sending events about the status of the computation within the TEE. The agent is found inside the VM (TEE), and each computation within the TEE has its own agent. When a computation run request is sent from from the manager, manager creates a VM where the agent is found and sends the computation manifest to the agent.
 
-## Agent events
+## Agent Events
 
 As the computation in the agent undergoes different operations, it sends events to the manager so that the user can monitor the computation from either the UI or other client. Events sent to the manager include computation running, computation finished, computation failed, and computation stopped.
 
-## gRPC connection between agent & manager
+## gRPC Connection Between Agent & Manager
```
Comment (SammyOina) [inline docs/agent.md:9]: there is no gRPC between agent and manager

File: docs/agent.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
 
-Agent is has a gRPC server which different clients such as CLI and manager can use to access different operations within the agent.
+Agent is has a gRPC server which different clients such as CLI can use to access different operations within the agent.
 
-## vsock connection between agent & manager
+## Vsock Bonnection Between Agent & Manager
```
Comment (SammyOina) [inline docs/agent.md:13]: fix typo

**Review by SammyOina — CHANGES_REQUESTED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**